dataactbroker/scripts/createGenerationJobs.py

```diff
@@ -0,0 +1,46 @@
+from dataactbroker.handlers.interfaceHolder import InterfaceHolder
+from dataactcore.models.jobModels import Submission, Job, JobDependency
+from dataactcore.models.baseInterface import databaseSession
+
+EXTERNAL_FILE_TYPES = ["award_procurement", "award", "awardee_attributes", "sub_award"]
+# Get all submission IDs
+interfaces = InterfaceHolder()
+submissionIds = interfaces.jobDb.session.query(Submission.submission_id).all()
+print(str(submissionIds))
+fileUpload = interfaces.jobDb.getJobTypeId("file_upload")
+validation = interfaces.jobDb.getJobTypeId("csv_record_validation")
+ready = interfaces.jobDb.getJobStatusId("ready")
+waiting = interfaces.jobDb.getJobStatusId("waiting")
+awardTypeId = interfaces.jobDb.getFileTypeId("award")
+awardProcTypeId =  interfaces.jobDb.getFileTypeId("award_procurement")
+externalIds = []
+for fileType in EXTERNAL_FILE_TYPES:
+    externalIds.append(interfaces.jobDb.getFileTypeId(fileType))
+# For each submission ID, check that all jobs are present and create any missing
+print("external IDs: " + str(externalIds))
+with databaseSession() as session:
```

Comment:
Yes the databaseSession function yields the session, and then calls _DB.close(), which closes the session and disposes the engine.